### DISCUSSION TITLE
Rendering SLS 'basessl' failed: mapping values are not allowed in this context

### DISCUSSION THREAD

**Discussion by milneyath** [Security-Onion-Solutions/securityonion#5291] [Unsupported Versions]
As title - getting this error when trying to run a Standalone Install on a VM...
Ran salt-call state.apply ca and salt-call state.highstate, as well as soup but all throw same error...
Anyone seen this? The actual ssl/init.sls file looks okay to me although I'm not super familiar with templating syntax...

This is latest version: 2.3.70-GRAFANA
Nothing in errors.log - sosetup.log is all standard until that last line: Rendering SLS 'basessl' failed: mapping values are not allowed in this context

**Comment by dougburks** [comment]
What options did you choose in Setup?

Did you choose to access your installation by IP, hostname, or other?

Did you make any manual changes after Setup?
